DataConnectors/JumpCloud Single Sign On/azuredeploy_JumpCloud_API_FunctionApp.json

```diff
@@ -0,0 +1,287 @@
+                        },
+                        {
+                            "name": "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING",
+                            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=', listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=core.windows.net')]"
+                        },
+                        {
+                            "name": "WEBSITE_CONTENTSHARE",
+                            "value": "[toLower(variables('FunctionName'))]"
+                        },
+                        {
+                            "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                            "value": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]"
+                        },
+                        {
+                            "name": "APPLICATIONINSIGHTS_CONNECTION_STRING",
+                            "value": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]"
+                        },
+                        {
+                            "name": "workspaceID",
+                            "value": "[parameters('WorkspaceID')]"
+                        },
+                        {
+                            "name": "workspaceKey",
+                            "value": "[parameters('WorkspaceKey')]"
+                        },
+                        {
+                            "name": "AzureSentinelTable",
+                            "value": "[parameters('AzureSentinelTable')]"
+                        },
+                        {
+                            "name": "JumpCloudApiToken",
+                            "value": "[parameters('JumpCloudApiToken')]"
+                        },
+                        {
+                            "name": "JumpCloudEventTypes",
+                            "value": "[parameters('JumpCloudEventTypes')]"
+                        },
+                        {
+                            "name": "JumpCloudStartTime",
+                            "value": "2020-08-04T00:00:00Z"
+                        },
+                        {
+                            "name": "JumpCloudUri",
+                            "value": "[parameters('JumpCloudUri')]"
+                        },
+                        {
+                            "name": "WEBSITE_RUN_FROM_PACKAGE",
+                            "value": "https://github.com/cabberley/JumpCloudSSO/raw/master/JumpCloudSSO.zip"
```

Comment:
Please replace value of WEBSITE_RUN_FROM_PACKAGE with https://aka.ms/sentinel-JumpCloudSSO-functionapp